packages/core/src/components/TextField/__tests__/__snapshots__/TextField.snapshot.test.js.snap

```diff
@@ -37,6 +37,7 @@ exports[`TextField renders correctly when disabled 1`] = `
         value=""
       />
       <div
+        aria-label=""
```

Comment:
it sound like the prop should be `undefined` if aria-label is empty, isn't it? and then the snapshots won't get an empty string aria label (and maybe in general it is better)

Reply:
Fixed